packages/dds/tree/src/tableSchema.ts

```diff
@@ -180,31 +202,34 @@ export namespace TableSchema {
 		const rowFields = {
 			id: schemaFactory.identifier,
 			cells: schemaFactory.map("Row.cells", cellSchema),
-		} as const satisfies Record<string, ImplicitFieldSchema>;
```

Comment:
We discovered that the `satisfies` clause here was actually resulting in different (and incorrect) typing to be applied to `rowFields`. This appears to be a TypeScript bug. But since this extra check isn't really necessary for type-safety in this function, it can be safely removed.